db/memtable.cc

```diff
@@ -523,6 +537,7 @@ class MemTableIterator : public InternalIterator {
   const MemTable::KeyComparator comparator_;
   MemTableRep::Iterator* iter_;
   bool valid_;
+  UnownedPtr<const SeqnoToTimeMapping> seqno_to_time_mapping_;
```

Comment:
Might be worth noting here that it's owned by the SuperVersion

Reply:
That's a good call. Thanks!